testsuites/listener/shared/client_sg/test_replication.py

```diff
@@ -1224,3 +1420,112 @@ def test_replication_with_session_cookie_short_ttl(setup_client_syncgateway_test
+    log_info("ls_url: {}".format(ls_url))
+    log_info("sg_admin_url: {}".format(sg_admin_url))
+    log_info("sg_url: {}".format(sg_url))
+
+    client = MobileRestClient()
+    client.create_database(url=ls_url, name=ls_db)
+    abc_channels = ["ABC"]
+    nbc_channels = ["NBC"]
+    num_of_docs = 10
+    client.create_user(sg_admin_url, sg_db, "autotest", password="password", channels=abc_channels)
+    session = client.create_session(sg_admin_url, sg_db, "autotest")
+    client.create_user(sg_admin_url, sg_db, "autotest2", password="password", channels=nbc_channels)
+    session1 = client.create_session(sg_admin_url, sg_db, "autotest2")
+
+    pull = client.start_replication(
+        url=ls_url,
+        continuous=True,
+        from_url=sg_url,
+        from_db=sg_db,
+        from_auth=session,
+        to_db=ls_db,
+    )
+
+    docs = client.add_docs(
+        url=sg_url,
+        db=sg_db,
+        number=num_of_docs,
+        id_prefix="seeded_doc",
+        generator="four_k",
+        channels=abc_channels,
+        auth=session
+    )
+
+    client.wait_for_replication_status_idle(ls_url, pull)
+    client.verify_docs_present(url=ls_url, db=ls_db, expected_docs=docs, timeout=240)
+    client.stop_replication(
+        url=ls_url,
+        continuous=True,
+        from_db=sg_db,
+        from_url=sg_url,
+        to_db=ls_db
+    )
+
+    # steps 5 and 6
+    for doc in docs:
+        log_info("doc is is {}".format(doc))
+        client.update_doc(url=sg_url, db=sg_db, doc_id=doc["id"], number_updates=1, attachment_name="sample_text.txt", delay=0.1, auth=session, channels=abc_channels)
+    client.update_docs(url=sg_url, db=sg_db, docs=docs, number_updates=1, delay=0.1, auth=session, channels=nbc_channels)
+    # Continue with step 7
+    pull = client.start_replication(
+        url=ls_url,
+        continuous=True,
+        from_url=sg_url,
+        from_db=sg_db,
+        from_auth=session,
+        to_db=ls_db,
+    )
+    client.wait_for_replication_status_idle(ls_url, pull)
+
+    # Step 8
+    updated_docs = client.update_docs(url=sg_url, db=sg_db, docs=docs, number_updates=1, delay=0.1, auth=session1, channels=abc_channels)
+    # Step 9
+    client.verify_docs_present(url=ls_url, db=ls_db, expected_docs=updated_docs, timeout=240)
+    for doc in updated_docs:
+        att = client.get_attachment(
+            url=ls_url,
+            db=ls_db,
+            doc_id=doc["id"],
+            attachment_name="sample_text.txt"
+        )
+        expected_text = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.\nUt enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.\nDuis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.\nExcepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
```

Comment:
Do not hard code, replace it with:
with open('sample_text.txt', 'r') as myfile:
    expected_text = myfile.read()

Reply:
done